openssl/src/asn1.rs

```diff
@@ -51,6 +57,32 @@ impl fmt::Display for Asn1TimeRef {
     }
 }
 
+impl Asn1TimeRef {
+    #[cfg(any(ossl102, ossl110))]
+    pub fn diff_from_epoch(&self) -> Result<TimeDiff, ErrorStack> {
+        Asn1Time::from_unix(0)?.diff(Some(self))
+    }
+
+    #[cfg(any(ossl102, ossl110))]
```

Comment:
These should be flagged on features like here: https://github.com/sfackler/rust-openssl/blob/master/openssl/src/ssl/mod.rs#L1415

Reply:
I think the link broke, what do you mean?

Reply:
https://github.com/sfackler/rust-openssl/blob/master/openssl/src/ssl/mod.rs#L1186-L1203